src/main/kotlin/no/nav/syfo/varsel/VarselDAO.kt

```diff
@@ -0,0 +1,85 @@
+package no.nav.syfo.varsel
+
+import org.springframework.jdbc.core.RowMapper
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Repository
+import java.sql.ResultSet
+import java.time.LocalDate
+import java.util.*
+
+@Repository
+class VarselDAO(
```

Comment:
Lurer på om denne klassen egentlig er mer et Repository. En DAO(Data Access Object) er ofte bare et enkelt data objekt som returneres fra et Repository.
```suggestion
class VarselRepository(
```

Reply:
Veldig enig Vetle! Jeg aner ikke hvorfor, men i alle våre apper har vi alltid kalt repositories for DAO. Synes det bør endres, så tenker vi gjør det :)